apps/total-typescript/src/components/desktop-exercise-navigator.tsx

```diff
@@ -7,7 +7,7 @@ import Link from 'next/link'
 import cx from 'classnames'
 import {Exercise} from '../lib/exercises'
 
-const ExerciseNavigator: React.FC<{
+const DesktopExerciseNavigator: React.FC<{
```

Comment:
this kept confusing me so I made name a bit more specific, also moved all of these to their own files to slim down the template file - was doing a lot of scrolling